tests/test_item.py

```diff
@@ -20,6 +22,46 @@ def get_example_item_dict(self) -> Dict[str, Any]:
             item_dict: Dict[str, Any] = json.load(f)
         return item_dict
 
+    def test_utc_datetimes_are_not_localized(self) -> None:
```

Comment:
To me, these tests should belong in a `test_utils.py::test_str_to_datetime` -- there's no item-specific functionality being checked, correct? This item test would then check the obvious regression case (tz="UTC"). That's just my test layout preference, though, not a required change (i.e. feel free to ignore).

Reply:
Yeah, good point. Putting them here actually helped me catch the fact that we weren't using `str_to_datetime` everywhere that we were parsing timestamps, but now that that is fixed we probably don't need these here as a regression test.

Reply:
Fixed in 6997000e40f248ea8779f53d5b41c3f535aa9a51